modules/dr-restoring-cluster-state.adoc

```diff
@@ -284,6 +284,57 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
 If the status is `Pending`, or the output lists more than one running etcd pod, wait a few minutes and check again.
 
 .. Repeat this step for each lost control plane host that is not the recovery host.
++
+[NOTE]
+====
+Perform the following step only if you are using `OVNKubernetes` Container Network Interface (CNI) plugin.
+====
++
+. Restart the Open Virtual Network (OVN) Kubernetes pods on all the hosts.
+
+.. Remove the northbound database (nbdb) and southbound database (sbdb). Access the recovery host and the remaining control plane nodes by using Secure Shell (SSH) and run the following command:
++
+[source,terminal]
+----
+$ sudo rm -f /var/lib/ovn/etc/*.db
+----
+
+.. Delete all master pods by running the following command:
```

Comment:
Is there any way to [avoid using the term "master"](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-conscious-language) here?

Reply:
Sure. We can use "control-plane" instead of "master". It is also followed here: https://docs.openshift.com/container-platform/4.11/post_installation_configuration/network-configuration.html#nw-ovn-ipsec-verification_post-install-network-configuration

Reply:
addressed